### ISSUE TITLE
how to get the json file

### ISSUE THREAD

**Issue by guotianli** [wkq-wukaiqi/Face-X-Ray#5]
hello. I have some mp4 videos,and  i want to use zhe code  by my data.
but  i am a newbee,  can you tell me  how to get the json file?

**Comment by wkq-wukaiqi** [comment]
The json files are from [FaceForensics++ repo](https://github.com/ondyari/FaceForensics/tree/master/dataset/splits). If you want to use your own data, you can try to build your own json file following the format of the official file.

**Comment by guotianli** [comment]
> The json files are from [FaceForensics++ repo](https://github.com/ondyari/FaceForensics/tree/master/dataset/splits). If you want to use your own data, you can try to build your own json file following the format of the official file.

Thank you, I see three json files. I'm just wondering how do I generate  json files of my own dataset。can you  tell me the step？or where  can i  study the knowledge

**Comment by wkq-wukaiqi** [comment]
> > The json files are from [FaceForensics++ repo](https://github.com/ondyari/FaceForensics/tree/master/dataset/splits). If you want to use your own data, you can try to build your own json file following the format of the official file.
> 
> Thank you, I see three json files. I'm just wondering how do I generate json files of my own dataset。can you tell me the step？or where can i study the knowledge

All I know is that it comes with the dataset, and if you're generating your own, the same numbering rules should work.

This json file consists of real video number pairs, such as `["071","054"]`, representing videos No.071 and No.054 exchanging faces with each other to generate two manipulated videos. Note that the training set, validation set, and test set should not have the same video.
